macros/parsers/parserMultiAnswer.pl

```diff
@@ -208,6 +72,15 @@ sub new {
 	}, $class;
 }
 
+#
+#  Set flags to be passed to individual answer checkers
+#
+sub setCmpFlags {
+	my ($self, $cmp_number, %flags) = @_;
+	die "Answer $cmp_number is not defined." unless defined($self->{cmp}[ $cmp_number - 1 ]);
+	$self->{cmp}[ $cmp_number - 1 ]->ans_hash(%flags);
+}
```

Comment:
In light of the comments by @drgrice1, perhaps this method should return `$self` so that you can chain the calls together.  E.g., 

``` perl
$ma = MultiAnswser("1/2", "sqrt(x-10)")->
         setCmpFlags(1, studentsMustReduceFractions => 1)->
         setCmpFlags(2, limits => [10, 15]);
```

and still have `$ma` be the MultiAnswer object.